donkeycar/parts/actuator.py

```diff
@@ -10,6 +10,18 @@
 from typing import Tuple
 
 import donkeycar as dk
+from donkeycar import utils
+from donkeycar.parts.kinematics import differential_steering
```

Comment:
Could it be that you missed to check in that module - this seems to cause the break of 3 tests: test_actuator, test_template, test_web_socket.

Reply:
Thanks for catching this.  the import of differential_steering is not necessary; in the branch I am picking from it have been move to the kinematics parts, but not in this branch.  I've removed the import.